src/client/client.ts

```diff
@@ -131,7 +131,12 @@ socket.addEventListener('message', async ({ data }) => {
       }
       break
     case 'full-reload':
-      location.reload()
+      if (
```

Comment:
I don't think this is correct - this would cause normal js-caused reloads to fail.

Reply:
Should only check `location.pathname` if `path` ends with `.html`.

Reply:
> Should only check `location.pathname` if `path` ends with `.html`.

If we change `b.html`, but the client path is `a.html`,it should not reload page.
Can I add other-type for handle js-caused reload.eg`js-full-reload`.